.github/workflows/presubmit.yaml

```diff
@@ -264,20 +264,25 @@ jobs:
         df -h
   docker-image-check:
     runs-on: ubuntu-22.04
+    needs: [build-debian-package]
     steps:
     - name: Free disk space
       uses: jlumbroso/free-disk-space@v1.3.1
       with:
         tool-cache: true
     - name: Checkout repository
       uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
-    - name: Install bazel
-      run: sudo bash tools/buildutils/installbazel.sh
+    - name: Download cuttlefish debs
+      uses: actions/download-artifact@v4
+      with:
+        name: debs
+        github-token: ${{ github.token }}
     - name: Run check
       shell: bash
       run: |
+        tar -xvf debs.tar && mv android-cuttlefish/cuttlefish*.deb .
         # TODO(b/420987840): Reuse deb packages created in `build-debian-package` job.
```

Comment:
Thanks for reusing the already built packages.

Please update b/420987840 and remove the TODO line.

Reply:
Updated the bug, and uploaded PR for removing TODO PR#1334

Thanks!